spec/conformance/index.md

```diff
@@ -77,3 +108,166 @@ completing the launch.
+
+{% include StructureDefinition-oauth-uris-inline.html %}
+
+<!-- =======    end inline view of extension ============== -->
+
+
+<!--
+### Publishing Additional Conformance Details
+
+A SMART on FHIR server should also describe which *optional* SMART core set of capabilities is it supports by declaring it within the CapabilityStatement [capabilities extension](#capabilities-extension) on the `rest.security` element (or, when using FHIR DSTU2, the
+`Conformance.rest.security` element).
+
+
+=======  inline view of extension ==============
+
+#### SMART Capabilities Extension
+
+Full StructureDefinition: [STU3](/StructureDefinition-extension-smart-capabilities.html), [DSTU2](/todo.html)
+
+{%raw%}{% include StructureDefinition-extension-smart-capabilities-inline.html %}{%endraw%}
+
+ =======    end inline view of extension ==============
+
+-->
+
+### Example
+
+{% include cs-example.md %}
+
+( for a complete example see the [CapabilityStatement Example](todo.html) )
+
+## FHIR Authorization Endpoint and Capabilities Discovery using a Well-Known Uniform Resource Identifiers (URIs)
+{: #using-well-known}
+
+As an alternative to using a FHIR CapabilityStatement, the authorization endpoints accepted by a FHIR resource server can be exposed as a Well-Known Uniform Resource Identifiers (URIs) [(RFC5785)][well-known] JSON document.
+
+FHIR endpoints requiring authorization MUST serve a JSON document at the location formed by appending `/.well-known/fhir-authorization-configuration` to their base URL.
+Contrary to RFC5785 Appendix B.4, the `.well-known` path component may be appended even if the FHIR endpoint already contains a path component.
+
+### Request
+
+Sample requests:
+
+#### Base URL "fhir.ehr.io"
+
+```
+GET /.well-known/fhir-authorization-configuration HTTP/1.1
+Host: fhir.ehr.io
+```
+
+#### Base URL "www.ehr.io/apis/fhir"
+
+```
+GET /apis/fhir/.well-known/fhir-authorization-configuration HTTP/1.1
+Host: www.ehr.io
+```
+
+### Response
+
+A JSON document must be returned using the `application/json` mime type.
+
+#### Metadata
+
+- `authorization_endpoint`: **REQUIRED**, URL to the OAuth2 authorization endpoint.
+- `token_endpoint`: **CONDITIONAL**, URL to the OAuth2 token endpoint, required unless the _implicit_ grant flow is used.
+- `token_endpoint_auth_methods`: **OPTIONAL**, array of client authentication methods supported by the token endpoint. The options are "client_secret_post" and "client_secret_basic".
+- `registration_endpoint`: **OPTIONAL**, if available, URL to the OAuth2 dynamic registration endpoint for this FHIR server.
+- `scopes_supported`: **RECOMMENDED**, array of scopes a client may request. See [scopes and launch context][smart-scopes].
+- `response_types_supported`: **RECOMMENDED**, array of OAuth2 `response_type` values that are supported
+- `manage_endpoint`: **RECOMMENDED**, URL where an end-user can view which applications currently have access to data and can make adjustments to these access rights.
```

Comment:
To keep consistency with verb tense in this SMART discovery json file, would say `management_endpoint`.